flask_testing/utils.py

```diff
@@ -453,7 +454,7 @@ def get_server_url(self):
         """
         Return the url of the test server
         """
-        return 'http://localhost:%s' % self._port_value.value
+        return 'http://{}:{}'.format(self._configured_host, self._port_value.value)
```

Comment:
To pass python2.6 testing, line should be apparently something more like

    return 'http://{hostname}:{port}'.format(
        hostname=self._configured_host,
        port=self._port_value.value,
    )